#include "main.hpp"

int main(int argc, char* argv[]) {

	try {



	}
	catch (std::exception& e) {

		std::cout << e.what() << std::endl;

		return 1;
	}

    return 0;
}